p2p/message.go

```diff
@@ -361,3 +369,29 @@ func readHash(r io.Reader) (common.Hash, error) {
 	return common.Hash(h), nil
 
 }
+
+type TransactionMessage []byte
```

Comment:
I think this should be redefined as `[]Extrinsic`. Not sure if we have that type defined anywhere, but right now you aren't treating the transactions as separate elements per the spec.

Reply:
I put an Extrinsic type in common/types.go btw

Reply:
I've refactored this to use []Extrinsic